libraries/ms-drivers/src/adt7476a_fan_controller.c

```diff
@@ -10,6 +9,58 @@
 #include "interrupt.h"
 #include "soft_timer.h"
 
+#ifdef MPXE
+#include <stdlib.h>
+#include "adt7476a.pb-c.h"
+#include "store.h"
+#include "stores.pb-c.h"
+
+static MxAdt7476aStore s_store = MX_ADT7476A_STORE__INIT;
+static uint32_t s_speed[NUM_ADT_PWM_PORTS] = { 0 };
+static bool s_status[NUM_ADT_PWM_PORTS] = { 0 };
```

Comment:
it seems like whenever we set `s_speed` or `s_status` we export the store right after. Instead can we remove these arrays and just set the status directly in s_store?

Reply:
Sorry? what do you mean about that? You mean just set `s_store.speed[0]` directly and call the store export after each function?

Reply:
yeah, do you think that would significantly reduce legibility? seems like an ok cut to me

Reply:
sure